cvat/apps/webhooks/views.py

```diff
@@ -39,14 +39,17 @@
     ),
     update=extend_schema(
         summary="Method updates a webhook by id",
-        responses={"200": WebhookWriteSerializer},
+        request=WebhookWriteSerializer, # check WebhookWriteSerializer.to_representation
+        responses={"200": WebhookReadSerializer},
     ),
     partial_update=extend_schema(
         summary="Methods does a partial update of chosen fields in a webhook",
-        responses={"200": WebhookWriteSerializer},
+        request=WebhookWriteSerializer, # check WebhookWriteSerializer.to_representation
```

Comment:
What does this comment mean?

Reply:
The current idea of serializers is that we have a Read serializer for responses and a Write serializer for requests. However, it is not expected by Django (`drf-rw-serializers` is expected to help with this). Because of this we have a switch in `get_serializer_class()` for modifying methods, but it must confuse the reader, how a WriteSerializer in `get_serializer_class()` could return a ReadSerializer in the response. Maybe, the comment should be better moved to the response line to be more clear.